common/logger/src/libra_logger.rs

```diff
@@ -18,11 +18,13 @@ use std::{
     io::Write,
     sync::{
         mpsc::{self, Receiver, SyncSender},
-        Arc,
+        Arc, RwLock,
     },
     thread,
 };
 
+const LIBRA_LOG: &str = "LIBRA_LOG";
+const RUST_LOG: &str = "RUST_LOG";
```

Comment:
So this is wishful thinking on my part. Our new logging system is sufficiently different from the `log` crate that I thought that it would make things clearer that the filtering is scoped to libra crates by having a LIBRA_LOG var. If you don't think it provides any value we can just remove it. At the moment it does the same thing as RUST_LOG

Reply:
I just dropped it.